plugins/view-resources/src/viewOptions.ts

```diff
@@ -52,7 +52,7 @@ function setStore (key: string, options: ViewOptions): void {
 }
 
 function _getViewOptions (viewlet: Viewlet, viewOptionStore: Map<string, ViewOptions>): ViewOptions | null {
-  const key = makeViewOptionsKey(viewlet._id, viewlet.variant)
```

Comment:
I think we need to check if new key exists use it, save to it, but reuse old value if possible to preserver current user values.

Reply:
Added this check